src/sdk/pynni/tests/test_compressor.py

```diff
@@ -138,7 +138,6 @@ def test_torch_fpgm_pruner(self):
         masks = pruner.calc_mask(layer, config_list[0], if_calculated=torch.tensor(0))
         assert all(torch.sum(masks['weight'], (1, 2, 3)).numpy() == np.array([45., 45., 45., 45., 0., 0., 45., 45., 45., 45.]))
 
-        pruner.update_epoch(1)
```

Comment:
what is the reason of removing this line?

Reply:
We circumvent wrapper to directly call cal_mask in UT.  FPGM pruner call wrapper in update epoch.